test/e2e/framework/clusterpropagationpolicy.go

```diff
@@ -46,6 +47,15 @@ func RemoveClusterPropagationPolicy(client karmada.Interface, name string) {
 	})
 }
 
+// RemoveClusterPropagationPolicyIfExist delete ClusterPropagationPolicy with karmada client if exits.
+func RemoveClusterPropagationPolicyIfExist(client karmada.Interface, name string) {
```

Comment:
Can we just modify the RemoveClusterPropagationPolicy method? I looked at where the method is called, and it don't care if the error is not found.